readme/installation.md

```diff
@@ -0,0 +1,107 @@
+# Installation
+
+Below are installation instructions for certain distributions as well as
+<a href="#manual-installation">Manual installation</a> instructions.
+
+We welcome contributors to maintain missing distributions.
+
+Please note that distributions may lag with the latest bug fixes.  To
+get the latest bug fixes, use the <a href="#manual-installation">Manual
+installation</a> method.
```

Comment:
"lag", I have never seen that word used in that context

Reply:
> "more or less" is unfortunately incorrect because it is stating that a distribution may never have current packages.

Which is why my proposal says "distributions **may** provide".

> I don't think we want to rewrite based on exposure to language.

I still don't like using lag like this. 

Please use "Please note that distributions may provide more or less outdated packages."